index.js

```diff
@@ -166,6 +169,11 @@ export const Rels = {
 	},
 	Users: {
 		canonicalUser: 'https://users.api.brightspace.com/rels/canonical-user'
+	},
+	UserProgress: {
+		Outcomes: {
+			activities: 'https://user-progress.api.brightspace.com/rels/outcome-activities'
```

Comment:
```suggestion
			activities: 'https://user-progress.api.brightspace.com/rels/outcome-activities',
			feedback: 'https://user-progress.api.brightspace.com/rels/feedback'
```
Going to be using this rel soon as well